source/common/http/http3/conn_pool.h

```diff
@@ -147,20 +149,12 @@ class Http3ConnPoolImpl : public FixedHttpConnPoolImpl {
 private:
   friend class Http3ConnPoolImplPeer;
 
-  // Returns the most recent crypto config from host_;
-  std::shared_ptr<quic::QuicCryptoClientConfig> cryptoConfig();
-
   // Store quic helpers which can be shared between connections and must live
```

Comment:
Should the comment be updated?  Maybe just call out we're latching quic helpers shared across the cluster?